docs/use_local_hyperledger.md

```diff
@@ -0,0 +1,102 @@
+#Use Local Hyperledger Network:
+
+### Creating a Local Hyperledger Network
+There is a convenient docker-compose image that will get you a network very quickly.
+
+1. Follow the docker compose [Setup Instructions](https://hub.docker.com/r/ibmblockchain/fabric-peer).
+1. Make sure your network is alive and reachable by testing the HTTP chain endpoint. To do this open your browser and browse to the peer.
+	- If you are running Windows with docker-toolbox then click http://192.168.99.100:7050/chain
+	- If you are running Linux/OS X/Windows 10 with native docker then click [http://localhost:7050/chain](http://localhost:7050/chain)
+	- If you changed the default port for peer 0 then you will need to edit the URL above to use that port instead of `7050`.
+1. You should see a response like:
+
+	```json
+	{
+		"height": 1,
+		"currentBlockHash": "lJ5dfqGBmhpkn1yHgbpbLnK9GEzrzsAnCm0AJZCIr0GaYznWDCt7j9yC09fGUe2MNXS+HEooKBbajHb+T40kIg==",
+		"previousBlockHash": "UYTfnosVy6PqW59Gs4roQTLZ5av/t8sMrkWDKetAwFzoueZ3SkIcW6qPVLQPHuxCJO17AxLYsjzmYNN1fNtwFg=="
+	}
+	```
+
+	- It will not be identical, but as long as you see some JSON response things are good and you can continue
+	- If you get a timeout or some other error message then your network is not yet running or you are not entering the correct URL.
+
+
+### Finished
+The network is all setup. 
+
+Next we need to **copy the peer data and pass it to our demo node.js application**.
+This is done by editing the `mycreds.json` file which lives in the root of the marbles app.
+
+All we must do is edit the file with information about your network.
+If you want more details of setup options then take a look at the [SDK's documentation](https://github.com/IBM-Blockchain/ibm-blockchain-js).
```

Comment:
Could just drop this.  We don't support `ibm-blockchain-js` anymore, and this demo doesn't use it.